Dockerfile

```
@@ -14,6 +14,9 @@ COPY packages /app/source/packages
 COPY scripts /app/source/scripts
 COPY types /app/source/types
 
+# Check to see if distutils is installed because python 3.12 removed it
+RUN python -c "import distutils" || apk update && apk add py3-setuptools
```

```suggestion
RUN python -c "import distutils" || apk --no-cache add py3-setuptools
```

If this works I'd try that, otherwise I think we end up adding package caches laying around in the image.